File: V3P3/V3P3.go
package main

import "fmt"

func main() {
	arr := []int{1, 2, 2, 3, 4, 4, 5, 5, 5, 7}

	output := []int{}

	for i := 0; i < len(arr); i++ {
		encontrado := false

		for j := 0; j < len(output); j++ {
			if arr[i] == output[j] {
				encontrado = true
			}
		}

		if !encontrado {
			output = append(output, arr[i])
		}
	}

	for _, x := range output {
		fmt.Println(x)
	}
}

File: V3P3/V3P3.rs
fn main() {
    let arr = [1, 2, 2, 3, 4, 4, 5, 5, 5, 7];
    let mut output = Vec::new();
    let mut encontrado;
    for i in 0..arr.len() {
        encontrado = false;
        for j in 0..output.len() {
            if arr[i] == output[j] {
                encontrado = true;
            }
        }
        if encontrado == false {
            output.push(arr[i])
        }
    }
    for x in &output {
        println!("{}", x);
    }
}
